elodie/filesystem.py

```diff
@@ -213,13 +218,38 @@ def process_file(self, _file, destination, media, **kwargs):
             shutil.move(_file, dest_path)
             os.utime(dest_path, (stat.st_atime, stat.st_mtime))
         else:
-            shutil.copy2(_file, dest_path)
+            shutil.copy(_file, dest_path)
+            self.set_date_from_filename(dest_path)
 
         db.add_hash(checksum, dest_path)
         db.update_hash_db()
 
         return dest_path
 
+    def set_date_from_filename(self, file):
+        """ Set the modification time on the file base on the file name.
+        """
+    
+        date_taken = None
+        file_name = os.path.basename(file)
+        # Initialize date taken to what's returned from the metadata function.
+        # If the folder and file name follow a time format of
+        #   YYYY-MM/DD-IMG_0001.JPG then we override the date_taken
```

Comment:
Any reason to use this format of `YYYY-MM/DD...`? I'd prefer if any assumptions were more general or in the worst case configurable.

I had patched the code to read in my own folder/filename format when I imported my photos; so I understand the rationale but wouldn't want that to bleed into the GitHub source.